web/portal/brand/src/entities/Company/Form.tsx

```diff
@@ -64,10 +68,12 @@ const Form = (props: EntityFormProps): JSX.Element => {
         'currency',
       ],
     },
-    edit && {
-      legend: _('Retail specific'),
-      fields: ['routingTagIds', 'codecIds'],
-    },
+    edit &&
+      !isVpbx &&
+      !isResidential && {
```

Comment:
`(wholesale || retail) && ...` would be better because if we add a new type in the future this will not fail